ocs_ci/deployment/deployment.py

```diff
@@ -3444,6 +3449,105 @@ def deploy(self):
         else:
             self.enable_managed_serviceaccount()
 
+    @retry(ACMObservabilityNotEnabled, tries=10, delay=30)
+    def check_observability_status(self):
+        """
+        Check observability status
+        Returns (bool): True or False
```

Comment:
It doesn't return True or False, it raises ACMObservabilityNotEnabled. Which logic did you actually choose?

Reply:
thanks, it was a stale doctext, fixed it